ui/app/AppLayouts/Chat/ChatColumn/ChatComponents/ChatCommandModal.qml

```diff
@@ -71,6 +82,9 @@ ModalPopup {
                 anchors.top: separator.bottom
                 anchors.topMargin: 10
                 width: stack.width
+                onSelectedRecipientChanged: {
+                    addressRequiredValidator.address = root.isRequested ? selectFromAccount.selectedAccount.address : selectRecipient.selectedRecipient.address
+                }
```

Comment:
We need to wait until `RecipientSelector` has determined its `selectedRecipient` before we can assign the recipient's address to the `addressRequiredValidator`.